### ISSUE TITLE
Azure DevOps Docker push: unknown flag: --threads

### ISSUE THREAD

**Issue by vaceslav** [jfrog/artifactory-vsts-extension#414]
### Describe the bug

Since today all builds in our Azure DevOps envs are broken

Error: **unknown flag: --threads**

### Current behavior

```
2023-05-10T08:35:48.4481265Z Login Succeeded
2023-05-10T08:35:53.2246683Z unknown flag: --threads
2023-05-10T08:35:53.2247142Z See 'docker push --help'.
2023-05-10T08:35:53.2258478Z exit status 125
2023-05-10T08:35:53.2306725Z ##[error]Error: Command failed: /opt/hostedtoolcache/jf/2.37.1/x64/jf docker push "replaced" --build-name="replaced" --build-number="replaced" --threads=3 --server-id="replaced"
2023-05-10T08:35:53.2314364Z Executing JFrog CLI Command:
2023-05-10T08:35:53.2314946Z /opt/hostedtoolcache/jf/2.37.1/x64/jf c remove "replaced" --quiet
2023-05-10T08:35:53.2467186Z ##[section]Finishing: Push image with image tag
```

My task looks like:

```
- task: JFrogDocker@1
            displayName: Push image with image tag
            inputs:
              command: 'Push'
              skipLogin: false
              artifactoryConnection: 'replaced'
              targetRepo: 'replaced'
              imageName: 'replaced'
              collectBuildInfo: true
              buildName: 'replaced'
              buildNumber: '$(imageTag)'
```

### Azure DevOps extension name and version

JFrogDocker@1

### JFrog CLI version

2.37.1

### Operating system type and version

Azure DevOps -> Ubuntu latest

**Comment by sverdlov93** [comment]
Hi @vaceslav 
Thanks for reporting this issue.
Our apologies for the inconvenience caused by this bug.
We believe that the issue was fixed on [jfrog-azure-devops-extension 2.6.1](https://github.com/jfrog/jfrog-azure-devops-extension/releases/tag/2.6.1)
Looking forward to your feedback on it.

**Comment by vaceslav** [comment]
Thank you.
I can confirm it works now.
